suzieq/engines/pandas/network.py

```diff
@@ -75,14 +76,34 @@ def _find_address(self, addr: str, **kwargs) -> pd.DataFrame:
 
         vlan = kwargs.pop('vlan', '')
         vrf = kwargs.pop('vrf', '')
+        columns = kwargs.get('columns', ['default'])
+        cols = self.schema.get_display_fields(columns)
 
         # Convert Cisco-style MAC address to standard MAC addr format,
         # and lowercase all letters of the alphabet
         addr = convert_macaddr_format_to_colon(addr)
 
         addr_df = self._find_addr_arp(addr, **kwargs)
         if addr_df.empty:
-            return addr_df
+            # Is this a locally attached interface IP to a device we're polling
+            df = self._get_table_sqobj('address') \
+                .get(vrf=vrf, address=[addr], columns=['*'],
+                     **kwargs)
+            if df.empty:
+                return addr_df
+
+            df = df.drop(columns=['ipAddressList', 'ip6AddressList',
+                                  'state', 'type'],
+                         errors='ignore')
+            df['ipAddress'] = df.ipAddress.apply(
+                lambda x: ', '.join([y.split('/')[0] for y in x]))
+
+            df['bondMembers'] = ''
+            df['type'] = 'interface'
+            df['l2miss'] = False
+
+            return df[cols]
```

Comment:
I think if one of the network columns is not in address, this line will fail

Reply:
Fixed